server/resources/abstract_resource.py

```diff
@@ -19,7 +19,6 @@ def __init__(self, resourceName, rootDirName):
         Resource.__init__(self)
         self.resourceName = resourceName
         self.rootDirName = rootDirName
-        self.route('GET', ('clear',), self.clear)
```

Comment:
This was meant for testing/debugging purposes. It's an admin only op. That said, I have no objection to removing it.

Reply:
This belongs to the same category as "fakeRun". It's useful for testing, but shouldn't be a part of public API.